scripts/asm_humidity_correction.py

```
@@ -0,0 +1,388 @@
+#!/usr/bin/env python3
+"""
+ASM Humidity Correction Script
+Re-audits and corrects all humidity values in cropConfigs to represent ASM percentages
+directly from scientific sources, not absolute humidity.
+
+Author: JXCT Soil Sensor Project
+Date: 2024
+"""
+
+import re
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)